src/screens/OtherProfile/index.js

```diff
@@ -614,13 +622,13 @@ const OtherProfile = () => {
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    backgroundColor: colors.white
+    backgroundColor: COLORS.white
   },
   content: {
     flexDirection: 'column',
     padding: 20,
     borderBottomWidth: 1,
-    borderBottomColor: '#E0E0E0'
+    borderBottomColor: COLORS.lightgrey
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color and border color properties have been updated to use the new `COLORS` object from the `theme` module. This is a good practice for maintaining consistency across the application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->